packages/suite/src/views/wallet/coinmarket/common/CoinmarketTag.tsx

```diff
@@ -19,8 +19,9 @@ const Tag = styled.div`
 
 interface CoinmarketTagProps {
     tag?: string;
+    className?: string;
 }
 
-export const CoinmarketTag = ({ tag }: CoinmarketTagProps) => (
-    <TagRow>{tag && <Tag>{tag}</Tag>}</TagRow>
+export const CoinmarketTag = ({ tag, className }: CoinmarketTagProps) => (
+    <TagRow className={className}>{tag && <Tag>{tag}</Tag>}</TagRow>
```

Comment:
Do you use `className` anywhere? I can't find it.

Reply:
It is not used directly, but without it the re-styling does not work - it looks like it is helping pass does the styles of my custom tag styled component. This solution was found in some stackoverflow issue.